canvas_modules/common-canvas/src/common-properties/properties-main/properties-main.jsx

```diff
@@ -570,18 +570,25 @@ class PropertiesMain extends React.Component {
 					{editorForm}
 				</PropertiesModal>);
 			}
+
+			let propertiesSizeClassname = `properties-${this.state.editorSize}`;
+
+			const overrideSize = this._getOverrideSize();
+			let overrideStyle = null;
+			if (overrideSize !== null) {
+				if (this.state.editorSize === Size.SMALL) {
```

Comment:
Why do we need this check?  Can't we juts always set this for overrideSize?  if there is a specific reason for this check then we should have a comment here.

Reply:
Since the issue was arising due to `properties-small` classname in below code.

https://github.com/elyra-ai/canvas/blob/b8c2388c921398c4e4aecc9596dd45fa078a570b/canvas_modules/common-canvas/src/common-properties/components/title-editor/title-editor.scss#L42

I have set custom classname only when editor size is small.


https://github.com/user-attachments/assets/a2100ffa-3e3a-4189-8be9-d71a31c5f4e1

If I remove that condition classname won't switch.


https://github.com/user-attachments/assets/b3f4eb53-0593-4a42-9304-39705b0fe276

Reply:
Sure. I have updated the code and also tests to include new classname `properties-custom-size`.